### DISCUSSION TITLE
@virtual tag for documentation comments

### DISCUSSION THREAD

**Discussion by azur-wolve** [godotengine/godot-proposals#10638] [Editor]
Let documentation comments have a `@virtual` tag:

![image](https://github.com/user-attachments/assets/a1cf557f-666b-48e2-9d14-1899cc7d4281)

 or 

![image](https://github.com/user-attachments/assets/1766f62f-0df6-4ae9-9714-4cc4d10abc45)

would mark it as virtual
![image](https://github.com/user-attachments/assets/626a807a-d189-461e-a65c-88c837207689)

**Comment by Mickeon** [comment]
This would probably be better accomplished with an annotation that actually denotes the method as virtual. Although, in GDScript it would not really mean much. It would mostly be there for visual flair and to display appropriate warnings.

See https://github.com/godotengine/godot-proposals/issues/1631 and https://github.com/godotengine/godot-proposals/issues/10500#issuecomment-2298614390

**Reply by azur-wolve** [reply]
yes exactly
its for documenting your own codebase

as it grows its hard to remember what was meant to be virtual and what private lets say

`@virtual func foo()`
and
`@override func foo()`
make more sense
